MediaBrowser.Providers/Lyric/LrcLyricProvider.cs

```diff
@@ -0,0 +1,189 @@
+            .Where(x => x.TimeTags.Count == 0)
+            .Where(x => x.Text.StartsWith('[') && x.Text.EndsWith(']'))
+            .Select(x => x.Text)
+            .ToList();
+
+        foreach (string metaDataRow in metaDataRows)
+        {
+            int colonCount = metaDataRow.Count(f => (f == ':'));
+            if (colonCount == 0)
+            {
+                continue;
+            }
+
+            string[] metaDataField = metaDataRow.Split(':', 2, StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+            string metaDataFieldName = metaDataField[0][1..].Trim();
+            string metaDataFieldValue = metaDataField[1][..^1].Trim();
+
+            fileMetaData.Add(metaDataFieldName, metaDataFieldValue);
+        }
+
+        if (sortedLyricData.Count == 0)
+        {
+            return null;
+        }
+
+        List<LyricLine> lyricList = new();
+
+        for (int i = 0; i < sortedLyricData.Count; i++)
+        {
+            var timeData = sortedLyricData[i].TimeTags.First().Value;
+            if (timeData is null)
+            {
+                continue;
+            }
+
+            long ticks = TimeSpan.FromMilliseconds(timeData.Value).Ticks;
+            lyricList.Add(new LyricLine(sortedLyricData[i].Text, ticks));
+        }
+
+        if (fileMetaData.Count != 0)
+        {
+            // Map metaData values from LRC file to LyricMetadata properties
+            LyricMetadata lyricMetadata = MapMetadataValues(fileMetaData);
+
+            return new LyricResponse { Metadata = lyricMetadata, Lyrics = lyricList };
+        }
+
+        return new LyricResponse { Lyrics = lyricList };
+    }
+
+    /// <summary>
+    /// Converts metadata from an LRC file to LyricMetadata properties.
+    /// </summary>
+    /// <param name="metaData">The metadata from the LRC file.</param>
+    /// <returns>A lyricMetadata object with mapped property data.</returns>
+    private static LyricMetadata MapMetadataValues(IDictionary<string, string> metaData)
+    {
+        LyricMetadata lyricMetadata = new();
+
+        if (metaData.TryGetValue("ar", out var artist) && !string.IsNullOrEmpty(artist))
+        {
+            lyricMetadata.Artist = artist;
+        }
+
+        if (metaData.TryGetValue("al", out var album) && !string.IsNullOrEmpty(album))
+        {
+            lyricMetadata.Album = album;
+        }
+
+        if (metaData.TryGetValue("ti", out var title) && !string.IsNullOrEmpty(title))
+        {
+            lyricMetadata.Title = title;
+        }
+
+        if (metaData.TryGetValue("au", out var author) && !string.IsNullOrEmpty(author))
+        {
+            lyricMetadata.Author = author;
+        }
+
+        if (metaData.TryGetValue("length", out var length) && !string.IsNullOrEmpty(length))
+        {
+            if (DateTime.TryParseExact(length, new string[] { "HH:mm:ss", "H:mm:ss", "mm:ss", "m:ss" }, null, DateTimeStyles.None, out var value))
```

Comment:
The formats array should be `private static readonly` (at the class level)

Reply:
`private static readonly string[] AcceptedTimeFormats => new[] { "HH:mm:ss", "H:mm:ss", "mm:ss", "m:ss" };`

I'm getting an error 'The modifier 'readonly' is not valid for this item.

Reply:
The code I just pushed used `private static` let me know if I need to adjust to something else of if I can resolve this.